packages/modal/src/Modal.tsx

```diff
@@ -0,0 +1,252 @@
+import React, { useCallback, SetStateAction, useRef } from 'react';
```

Comment:
When the content of the modal is larger than the viewport the modal content does not scroll. This is a tricky problem, may want to pair with @jetpacmonkey on this.